src/Castle.Services.Logging.NLogIntegration/Castle.Services.Logging.NLogIntegration.csproj

```diff
@@ -3,7 +3,7 @@
 	<Import Project="..\..\buildscripts\common.props"></Import>
 
 	<PropertyGroup>
-		<TargetFrameworks>net45;netstandard1.5</TargetFrameworks>
+		<TargetFrameworks>net45;netstandard1.5;netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
I suppose adding just `netstandard2.0` would be sufficient. I've included `netstandard2.1` mainly for consistency with the main library&mdash;I expect it'll be less confusing in the future to see the same list of target frameworks everywhere. (Same goes for the other logging integration projects.)